google_datastream/.terraform-docs.yml

```diff
@@ -6,22 +6,22 @@ content: |-
   ### Prework
   - Pick a new /29 network for Datastream to use. This is the datastream_subnet under Inputs below. Add the subnet to https://mozilla-hub.atlassian.net/wiki/spaces/SRE/pages/27920489/GCP+Subnet+Allocations for tracking
   - Create a CloudSQL Auth Proxy so Datastream can connect to Cloud SQL
-    - Doing this is outside the scope of these docs (that's convenient!), but see here for an example
-      - CloudSQL Auth Proxy Deployment: https://github.com/mozilla-it/webservices-infra/blob/main/moztodon/k8s/moztodon/templates/deployment-cloudsqlauthproxy.yaml
-      - CloudSQL Auth Proxy Service: https://github.com/mozilla-it/webservices-infra/blob/main/moztodon/k8s/moztodon/templates/service-cloudsqlauthproxy.yaml
+  - - Doing this is outside the scope of these docs (that's convenient!), but see here for an example
+  - - - CloudSQL Auth Proxy Deployment: https://github.com/mozilla-it/webservices-infra/blob/main/moztodon/k8s/moztodon/templates/deployment-cloudsqlauthproxy.yaml
+  - - - CloudSQL Auth Proxy Service: https://github.com/mozilla-it/webservices-infra/blob/main/moztodon/k8s/moztodon/templates/service-cloudsqlauthproxy.yaml
```

Comment:
Looks like this formatting will include all the bullet points at each indentation instead of just indenting things:

![image](https://github.com/mozilla/terraform-modules/assets/126706864/d72ba5c7-4b94-463a-bb14-8382c5ef5468)

^ from https://github.com/mozilla/terraform-modules/tree/fix-overly-restrictive-versions/google_datastream

Reply:
ugh